### PR TITLE
fixes bug 1264940 - validate date range on crashes per day

### PR DESCRIPTION
**Pull request by peterbe**
This just prevents people from using the date parameter wrong and triggering a internal server error.

### CODE REVIEW

File: webapp-django/crashstats/crashstats/forms.py [hunk 1]
```diff
@@ -235,6 +235,16 @@ def clean_v(self):
             )
         return versions
 
+    def clean(self):
+        cleaned_data = super(DailyFormBase, self).clean()
+        if cleaned_data.get('date_end') and cleaned_data.get('date_start'):
+            # Check the invariant. Make sure the start < end.
+            if cleaned_data['date_start'] > cleaned_data['date_end']:
+                raise forms.ValidationError(
+                    'date start > date end'
```
Comment (adngdb) [inline webapp-django/crashstats/crashstats/forms.py]: That's a pretty obscure error. Why not writing something a bit more explicit, like ``start date has to be lower than end date``?

**Comment by adngdb** [comment]
No tests for this?

File: webapp-django/crashstats/crashstats/forms.py (hunk 1 above)
Comment (peterbe) [inline webapp-django/crashstats/crashstats/forms.py]: I'll rewrite it to say "Start date greater than end date". I think that's more useful to whoever has to see this.

**Comment by peterbe** [comment]
> No tests for this? 

That's why reviews are so useful! :)

**Comment by adngdb** [comment]
r+

**Pull request merged**